packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -78,35 +80,83 @@ We will not make any special effort to provide forms support (i.e. form `type="d
+    -   Additional details: The content provided for the title and sub-title will be used as the label of the dialog for accessibility purposes.
+-   Subtitle
+    -   Included in initial styling pass: Yes
+    -   Rationale: The styling of the subtitle is straight-forward and there are no open questions associated with it.
+    -   Additional details: The content provided for the title and sub-title will be used as the label of the dialog for accessibility purposes.
+-   Warning message
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this part of the dialog yet, and there are a number of questions that need to be resolved prior to adding the messages:
+        -   Should the messages be arbitrary content provided by the user, or should we have an API around providing message(s) with a specified severity, icon, color, etc?
+        -   What, if any, is the overlap between these messages and a future `<nimble-banner>` component?
+        -   Do we allow multiple messages at the same time? If so, is there a maximum height the messages can consume within the dialog?
+-   Close button
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this component yet, and there are a number of questions that need to be resolved prior to adding the button:
+        -   Should the API allow a client to slot any arbitrary buttons, including a close button, or should we only support one single button that closes the dialog?
+        -   What control do we need to give over the close button? Do we need a way for the user to explicitly disable it?
+        -   Should there be any connection between the `prevent-dismiss` attribute on the dialog and the visibility or disabled state of the close button?
+-   Content
+    -   Included in initial styling pass: Yes
+    -   Rationale: This is critical for using a dialog
+    -   Additional details:
+        -   The content is a `flex` layout container that allows a clinet to easily place multiple items within the dialog.
+        -   Content will have a `flex-direction` of `column` so that content stacks vertically.
+        -   A `gap` will be provided so that that mutliple content elements have appropriate spacing between them.
+-   Footer buttons
+    -   Included in initial styling pass: Yes
+    -   Rationale: These buttons are required for interacting with the dialog.
+    -   Additional details:
+        -   The footer is a `flex` layout container that allows a client to easily place buttons at the bottom of the dialog.
+        -   If no content is slotted in the footer, the horizontal separator between the footer and content will be removed. The height of the footer will also be adjusted to align with the visual design spec.
+        -   If content is slotted in the footer, it will automatially be placed with the correct horizontal spacing between the vertical line and the bottom of the dialog.
```

Comment:
I could be missing something but this sentence is unclear to me. Is the intent that the dialog will specify both:
1. the horizontal spacing between the buttons 
2. the vertical spacing that separates the buttons from the line above and the bottom of the dialog below?

![image](https://user-images.githubusercontent.com/10500124/192563854-700bb089-ed33-4dea-a9f3-8c4ec73be101.png)

If so then we could change it to something like this:

```suggestion
        -   If content is slotted in the footer, it will automatically be placed with the correct vertical spacing between the horizontal line and the bottom of the dialog and with the correct horizontal spacing around content items.
```

Reply:
Apparently I was having trouble with vertical vs horizontal yesterday 🤦